### DISCUSSION TITLE
connect ETIMEDOUT error when testing SMTP configuration

### DISCUSSION THREAD

**Discussion by ColinMaudry** [requarks/wiki#3061] [Help / Questions]
Hello,

OS: Debian 10.1
Wiki.js: 2.5.170
Node.js: 12.20.1

Installed via Yunohost 4.1 on a Scaleway development instance.

I want to use an external SMTP provider to send emails. I tried with my Fastmail and Gandi account, I always get the same error when I click on "Send an email" (to test): 

> connect ETIMEDOUT [provider IP]:465.

When people create a local account, they don't receive the validation email.

The configuration is pretty straightforward, I follow the same instructions I used to successfully configure my email client (Thunderbird): https://docs.gandi.net/fr/gandimail/configuration_messagerie/index.html#compte-smtp

I have tried with and without SSL enabled (ports 465 and 25).

I have double checked the password.

Do I need to open ports on the host?

Thanks!

**Comment by ColinMaudry** [comment]
I have not enabled DKIM (don't know how).

**Comment by ColinMaudry** [comment]
I don't see anything in the NodeJS logs.

**Comment by ColinMaudry** [comment]
Same result with smtp.gmail.com.

**Comment by njyo** [comment]
@ColinMaudry, have you found a resolution? I'm trying to set up the same, but it doesn't seem to work…
Any advice is much appreciated. 🙂 
Thanks!

**Reply by ColinMaudry** [reply]
Sorry I don't use wiki.JS anymore. Good luck!

**Reply by njyo** [reply]
Thanks, @ColinMaudry! :)
